src/amcrest/ptz.py

```diff
@@ -220,6 +220,17 @@ def go_to_preset(self, action=None, channel=0, preset_point_number=1):
         )
         return ret.content.decode('utf-8')
 
+    def set_preset(self, action=None, channel=0, preset_point_number=1):
```

Comment:
I would add a `'start'` as the default value for `action`, like what I did when I added the `tour` method.

Reply:
> I would add a `'start'` as the default value for `action`, like what I did when
> I added the `tour` method.
I can't see the tour method in the ptz.py  :(  Could you please point?

Reply:
have default as start looks good to me too. The code looks sane to me.